volume/driver/overlay_linux.go

```diff
@@ -68,15 +68,10 @@ func (driver *OverlayDriver) DestroyVolume(path string) error {
 }
 
 func (driver *OverlayDriver) CreateCopyOnWriteLayer(path string, parent string) error {
-	ancestry, err := driver.ancestry(parent)
```

Comment:
Can we kill the `ancestry` method too? I don't see it used anymore (though GitHub search could be failing me).

Reply:
ooh it's used in the "remount"ing: 

https://github.com/concourse/baggageclaim/blob/dc362b1f055e4836dc2afb16639383c61cee637a/volume/driver/overlay_linux.go#L139-L142 

I didn't really look at it yet though

Reply:
(going to try it out here)

Reply:
yeahh ... we'll have to modify that too - and it turns out that we can't just go with the `parent` directly: the `mount(2)` syscalls will work, but the data *might* not show up, depending on the order of the mounts.

if it just turns out that in the iteration we get the `cow_cow` before `cow`, we'll be able to mount `cow_cow` just fine, but by mounting `cow` right after, it's as if you had "stacked", thus, `cow_cow` won't see the contents from `cow`.

it seems to me that we'd need to make sure that we mount parents first, and then cow_cow later